docs/source/workflows/design_workflows.rst

```diff
@@ -35,18 +35,13 @@ The following example demonstrates how to use the python SDK to register a workf
     # (i.e. why the workflow is valid/invalid)
     print(validated_workflow.status_info)
 
-.. warning::
-    
-    Registering DesignWorkflows using `project.workflows` is deprecated, please use `project.design_workflows` as described above instead.
-
-
 .. code:: python
```

Comment:
I think you should be able to remove this example, because the `.design_workflow` example already exists just above